src/components/instances/new/MetaLocal.js

```diff
@@ -50,7 +51,23 @@ function MetaLocal() {
 				setFormState({ error: 'usernames must have only letters, underscores, and hyphens' });
 			} else if (instance_name.length && user.length && pass.length && host.length && port.length) {
 				try {
-					const currentUser = await userInfo({ auth: { user, pass }, url, is_local: true, customer_id });
+					let loginNotSupported;
```

Comment:
I'm confused by what we are trying to do here. It looks like this was just copy and pasted from CardBackLogin, but that is where the actual login occurs, whereas I though this code was just checking to see if the user was already authenticated. And this logic largely existed to determine if login was supported and appropriately save the credentials, but `loginNotSupported` is even't used here (and shouldn't be); and it doesn't make sense to determine that here, I don't think. And if there is an existing `pass` then we should already have determined that basic auth is the way (not login).

Reply:
Transparently, I was trying to do a quick patch and didn't dive to far into the code 😅  to focus back on Fabric UI at the time. I'll properly fix it!